react/src/base/inputs/SprkCheckbox.stories.js

```diff
@@ -92,10 +94,14 @@ export const invalidCheckbox = () => (
       <SprkCheckboxItem>Checkbox Item 2</SprkCheckboxItem>
       <SprkCheckboxItem>Checkbox Item 3</SprkCheckboxItem>
     </SprkFieldset>
```

Comment:
Need aria-describedby in this checkbox example with the id of the error container.